config/locales/partials/data_overview/registration_info.en.yml

```diff
@@ -12,6 +12,7 @@ en:
             value:
               "true": "Yes"
               "false": "No"
+            change_link_suffix: "on a farm"
```

Comment:
According to AC#3, this should be "Will this operation be carried out on a farm"